### PR TITLE
Fix button label

### PR DESCRIPTION
**Pull request by blackbaud-johnly**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-modals?_sp.spa=pr-button-label-030121](https://developer.blackbaud.com/skyux-modals?_sp.spa=pr-button-label-030121)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-modals?_sp.spa=pr-button-label-030121](https://developer.blackbaud.com/skyux-modals?_sp.spa=pr-button-label-030121)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-modals?_sp.spa=pr-button-label-030121](https://developer.blackbaud.com/skyux-modals?_sp.spa=pr-button-label-030121)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-modals?_sp.spa=pr-button-label-030121](https://developer.blackbaud.com/skyux-modals?_sp.spa=pr-button-label-030121)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-modals?_sp.spa=pr-button-label-030121](https://developer.blackbaud.com/skyux-modals?_sp.spa=pr-button-label-030121)

File: src/app/docs/modal/modal-docs.component.html [hunk 1]
```diff
@@ -119,12 +119,14 @@
...
-          Use a <a href="https://developer.blackbaud.com/skyux/components/button">primary button</a> for the most important action that users take. Use secondary buttons for less important actions. Use a link button for the Cancel action.
-        </p>
-        <p class="sky-docs-text-block">
-          If a modal has only one action, you should generally use the <a href="https://developer.blackbaud.com/skyux/components/confirm">confirm component</a> instead.
-        </p>
+        <div class="sky-docs-text-block">
+          <p>
+            Button labels should clearly indicate the actions that occur when users select buttons. For example, use <strong>Save</strong> as the standard label for actions where users save information to the database, but use <strong>Submit</strong> or another alternative if you need to indicate that an action sends data somewhere else. Make sure to use the label that is most accurate for all contexts. For example, while <strong>Cancel</strong> and <strong>Close</strong> both indicate that actions close modals, <strong>Cancel</strong> more clearly indicates that the actions do not save changes to modals and should be used in most cases.
```
Comment (blackbaud-paulah) [inline src/app/docs/modal/modal-docs.component.html:124]:
```suggestion
            Button labels should clearly indicate the actions that occur when users select buttons. For example, use <strong>Save</strong> as the standard label for actions where users save information to the database, but use <strong>Submit</strong> or another alternative if you need to indicate that an action sends data somewhere else. Make sure to use the label that is most accurate for all contexts. For example, while <strong>Cancel</strong> and <strong>Close</strong> both indicate that actions close modals, <strong>Cancel</strong> more clearly indicates that the actions do not save changes to modals; <strong>Cancel</strong> should be used in most cases.
```

**Review by blackbaud-paulah — DISMISSED** [review]
Changing the sentence structure highlights the idea that Cancel should be used in most cases.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-modals?_sp.spa=pr-button-label-030121](https://developer.blackbaud.com/skyux-modals?_sp.spa=pr-button-label-030121)

**Review by blackbaud-paulah — APPROVED** [review]

**Review by Blackbaud-AdamFunderburk — APPROVED** [review]

**Pull request merged**